### PR TITLE
changed to english and styling some screens

### PR DESCRIPTION
**Pull request by Annangggg**

### CODE REVIEW

**Review by gentrithotii — APPROVED** [review]

File: src/screens/household/TasksOverviewScreen.tsx [hunk 1]
```diff
@@ -281,7 +282,7 @@ export default function TaskOverviewScreen({
             task.description,
             [
               {
-                text: "Se mer",
+                text: "See more",
```
Comment (OdiousCode) [inline src/screens/household/TasksOverviewScreen.tsx:285]: Details

File: src/screens/household/PendingApplicationsScreen.tsx [hunk 2]
```diff
@@ -124,48 +125,48 @@ export default function PendingApplicationScreen({
...
+                                    "Are you sure you want to remove the user?",
                                     [
                                       {
-                                        text: "Ja",
+                                        text: "Yes",
                                         onPress: () => {
                                           Alert.alert(
                                             profile.email +
-                                              ' har tagits bort ur hushåll "' +
+                                              ' have been removed from household "' +
```
Comment (OdiousCode) [inline src/screens/household/PendingApplicationsScreen.tsx:143]: has been

File: src/screens/household/PendingApplicationsScreen.tsx [hunk 3]
```diff
@@ -180,49 +181,49 @@ export default function PendingApplicationScreen({
...
                                     {
-                                      text: "Ja",
+                                      text: "Yes",
                                       onPress: () => {
                                         // Lägg in kod för att faktiskt se till att profilen lämnar hushåll här.
                                         allowUser(profile);
                                         Alert.alert(
                                           profile.email +
-                                            ' har lagts till i hushållet"' +
+                                            ' have been added to the household "' +
```
Comment (OdiousCode) [inline src/screens/household/PendingApplicationsScreen.tsx:201]: has been

File: src/screens/household/ProfileOverviewScreen.tsx [hunk 4]
```diff
@@ -212,43 +212,43 @@ export default function ProfileOverViewScreen({
...
                                             [
                                               {
-                                                text: "Ja",
+                                                text: "Yes",
                                                 onPress: () => {
                                                   // Lägg in kod för att faktiskt se till att profilen lämnar hushåll här.
 
                                                   Alert.alert(
-                                                    'Lämnat hushåll "' +
+                                                    'leave household "' +
```
Comment (OdiousCode) [inline src/screens/household/ProfileOverviewScreen.tsx:229]: Left household

File: src/screens/household/TasksOverviewScreen.tsx [hunk 5]
```diff
@@ -86,7 +87,7 @@ export default function TaskOverviewScreen({
     setVisible(false);
   };
   const handleOk = (newName: string) => {
-    Alert.alert("Bytt namn till", newName);
+    Alert.alert("Change name to", newName);
```
Comment (OdiousCode) [inline src/screens/household/TasksOverviewScreen.tsx:90]: Changed

File: src/screens/household/TasksOverviewScreen.tsx [hunk 6]
```diff
@@ -105,31 +106,31 @@ export default function TaskOverviewScreen({
...
+          Hello {activeProfile?.name}
         </Text>
-        <Text>
+        <Text style={styles.date}>
           {day}: {getCurrentDate()}
         </Text>
         {activeProfile?.role === "Admin" ? (
           <Pressable onPress={showDialog}>
-            <Text>Sysslor för {activeHousehold?.name}</Text>
+            <Text style={styles.date}>New task: {activeHousehold?.name}</Text>
```
Comment (OdiousCode) [inline src/screens/household/TasksOverviewScreen.tsx:117]: Tasks:

File: src/screens/household/TasksOverviewScreen.tsx [hunk 7]
```diff
@@ -105,31 +106,31 @@ export default function TaskOverviewScreen({
...
           </Pressable>
         ) : (
-          <Text style={{ fontSize: 14, fontWeight: "bold", marginBottom: 5 }}>
-            Sysslor för {activeHousehold?.name}
+          <Text style={{ fontSize: 14, fontWeight: "bold", marginBottom: 10 }}>
+            New task: {activeHousehold?.name}
```
Comment (OdiousCode) [inline src/screens/household/TasksOverviewScreen.tsx:121]: Tasks:

File: src/screens/household/TasksOverviewScreen.tsx [hunk 8]
```diff
@@ -105,31 +106,31 @@ export default function TaskOverviewScreen({
...
           </Text>
         )}
         <Dialog.Container visible={visible}>
-          <Dialog.Title>Namnbyte på hushåll</Dialog.Title>
-          <Dialog.Description>Vad vill du byta namnet till?</Dialog.Description>
+          <Dialog.Title>Name changes for household</Dialog.Title>
+          <Dialog.Description>What is your prefer name?</Dialog.Description>
```
Comment (OdiousCode) [inline src/screens/household/TasksOverviewScreen.tsx:126]: Change household name

New name?

**Review by OdiousCode — APPROVED** [review]

**Pull request merged**